tools/osbuild-depsolve-dnf

```diff
@@ -364,22 +840,23 @@ def solve(request, cache_dir):
     arguments = request["arguments"]
 
     with tempfile.TemporaryDirectory() as persistdir:
+        solver = globals()[solverClassName](request, persistdir, cache_dir)
```

Comment:
I'd change this to just do a comparison test and call the class. It's not like it is going to be expanded to support many (any!!?) more versions and I find this hard to read.